packages/logger/src/index.ts

```diff
@@ -18,7 +18,7 @@ type LoggerConstructorOptions = {
 class Logger {
   private readonly asyncOperationId: string | undefined;
   private readonly executions: string | undefined;
-  private readonly granules: string | undefined;
+  private readonly granules: string | Array<string> | undefined;
```

Comment:
It's actually
```suggestion
  private readonly granules: string | string[] | undefined;
```

Reply:
preferred though, both are correct?  https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#arrays

Reply:
Fair, but our existing code only uses `type[]` in typedefs, so I'd still argue consistency if you hadn't already resolved it. 🙂